packages/cli/src/swagger/specGenerator.ts

```diff
@@ -8,8 +9,17 @@ export abstract class SpecGenerator {
     return this.getSwaggerType(type);
   }
 
-  protected getOperationId(methodName: string) {
-    return methodName.charAt(0).toUpperCase() + methodName.substr(1);
+  protected buildOperationIdTemplate(inlineTemplate: string) {
+    handlebars.registerHelper('titleCase', (value: string) => (value ? value.charAt(0).toUpperCase() + value.slice(1) : value));
+    handlebars.registerHelper('replace', (subject: string, searchValue: string, withValue = '') => (subject ? subject.replace(searchValue, withValue) : subject));
```

Comment:
Ideally, we should doi this once, not every time we call `buildOperationIdTemplate`